definitions/variable/tag_electricity_input_types.yaml

```diff
@@ -243,31 +243,9 @@
       description: hydro, wind, solar, geothermal, ocean, and other renewable sources
         excluding bioenergy
 
-  - Hydro:
-      description: hydropower
-
-  - Hydro|Run of River:
-      description: run-of-river hydro power plants (i.e. without storage capacity)
-
-  - Hydro|Pumped Storage:
-      description: hydropower systems with short-term storage capacity
-        (including pumping turbines)
-      notes: This hydropower system has two water reservoirs at different elevations.
-        The definition follows the convention established
-        by the ENTSO-E transparency platform (https://transparency.entsoe.eu).
-
-  - Hydro|Reservoir:
-      description: hydropower systems with seasonal storage capacity
-        (possibly including pumping turbines)
-      notes: This definition follows the convention established
-        by the ENTSO-E transparency platform (https://transparency.entsoe.eu).
-
```

Comment:
You should keep Run of River here as it is not a storage unit (see definition)

Reply:
Yeah, you are right. I changed.